hyundai_kia_connect_api/Vehicle.py

```diff
@@ -109,6 +106,10 @@ class Vehicle:
 
     # EV fields (EV/PHEV)
 
+
+    ev_charge_limits_dc: typing.Union[int, None] = None
```

Comment:
the `typing` module must be imported

`import typing`

unit tests should pass after that